### PR TITLE
Add help command

### PR DESCRIPTION
**Pull request by dlsf**

### CODE REVIEW

**Comment by PeachesMLG** [comment]
Looks good, want me to leave thsi open while u add the page system or merge it now?

**Comment by dlsf** [comment]
I'll be adding the page system first

File: src/main/java/com/iridium/iridiumskyblock/commands/HelpCommand.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+     * Executes the command for the specified {@link CommandSender} with the provided arguments.
+     * Not called when the command execution was invalid (no permission, no player or command disabled).
+     *
+     * @param sender    The CommandSender which executes this command
+     * @param arguments The arguments used with this command. They contain the sub-command
+     */
+    @Override
+    public void execute(CommandSender sender, String[] arguments) {
+        Messages messages = IridiumSkyblock.getInstance().getMessages();
+        String header = messages.helpCommandHeader;
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/commands/HelpCommand.java:34]: This variable isnt needed

Comment (PeachesMLG) [reply src/main/java/com/iridium/iridiumskyblock/commands/HelpCommand.java:34]: commandTemplate should be removed aswell

File: src/main/java/com/iridium/iridiumskyblock/commands/HelpCommand.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    /**
+     * Executes the command for the specified {@link CommandSender} with the provided arguments.
+     * Not called when the command execution was invalid (no permission, no player or command disabled).
+     *
+     * @param sender    The CommandSender which executes this command
+     * @param arguments The arguments used with this command. They contain the sub-command
+     */
+    @Override
+    public void execute(CommandSender sender, String[] arguments) {
+        Messages messages = IridiumSkyblock.getInstance().getMessages();
```
[the remark this answers is not in the document]
Comment (dlsf) [reply src/main/java/com/iridium/iridiumskyblock/commands/HelpCommand.java:33]: I get your point but at the same time, it'd get so much longer when writing it out every time. It's up to you

**Pull request merged**